R/pkg/R/generics.R

```diff
@@ -1415,6 +1415,13 @@ setGeneric("spark.freqItemsets", function(object) { standardGeneric("spark.freqI
 #' @rdname spark.fpGrowth
 setGeneric("spark.associationRules", function(object) { standardGeneric("spark.associationRules") })
 
+#' @rdname spark.prefixSpan
+setGeneric("spark.prefixSpan", function(...) { standardGeneric("spark.prefixSpan") })
+
+#' @rdname spark.prefixSpan
```

Comment:
`findFrequentSequentialPatterns`?

Reply:
@viirya Thanks for your review. 
Do you mean to use ```findFrequentSequentialPatterns``` instead of ```spark.findFrequentSequentialPatterns```? I have a question about when we need to add ```spark``` in front of the method name. In FPGrowth, some methods have ```spark.```, e.g.  ```spark.freqItemsets```, ```spark.associationRules```. Some methods don't have ```spark.```, e.g. ```predict``` and ```write.ml``` .

Reply:
i think he meant the rdname - yes ok to have both in one rd

Reply:
OK thanks. I guess I will keep both spark.prefixSpan and spark.findFrequentSequentialPatterns in the same rd.